src/lua/oltp_common.lua

```diff
@@ -37,6 +37,8 @@ sysbench.cmdline.options = {
       {"Range size for range SELECT queries", 100},
    tables =
       {"Number of tables", 1},
+   num_table_splits =
```

Comment:
Instead of this approach, can we do:
1. Have an option `use_range_tables=true` which will use range tables instead of hash tables.
2. When range tables are used, we pre-split the tables. We can pre-split into 24 tablets using `table_size`. So split points will be `table_size / 24`, `2 * table_size / 24`, `3* table_size / 24` and so on.

For now, use_range_tables will be false by default. We could make it true in future.